src/RequestClientWrapper.ts

```diff
@@ -117,16 +112,9 @@ export default class RequestClientWrapper {
     if (options.headers === undefined) {
       options.headers = {};
     }
-    this.setSupabaseEnv(options.baseURL);
     if (this.isJwtToken) {
-      let accessToken = this.requestClientWrapperConfiguration.session;
-      const session = await this.checkSupabaseSession();
-      if (session) {
-        accessToken = session.access_token;
-      } else if (this.isJwtTokenExpired(accessToken)) {
-        accessToken = await this.refreshSupabaseSession();
-      }
-      options.headers['X-Channel-Ape-Jwt'] = accessToken;
+      const accessToken = this.requestClientWrapperConfiguration.session;
+      options.headers.Authorization = `Bearer ${accessToken}`;
     } else {
       options.headers['X-Channel-Ape-Authorization-Token'] = this.requestClientWrapperConfiguration.session;
```

Comment:
Can we log a warning here stating that this is a deprecated token and that you should switch to a new token since the old ones aren't supported anymroe? This could help us detect any lingering services in our stack that are potentially using this.